src/component/common.js

```diff
@@ -789,29 +874,17 @@ const ComponentHeader = ({
   name,
   version,
   ocmRepository,
+  initialVersionFilter,
   isLoading,
 }) => {
   const navigate = useNavigate()
 
   const searchParamContext = React.useContext(SearchParamContext)
 
-  const featureRegistrationContext = React.useContext(FeatureRegistrationContext)
-
   const [componentName, setComponentName] = React.useState(name)
   const [componentVersion, setComponentVersion] = React.useState(version)
-  const [versionFilter, setVersionFilter] = React.useState(VERSION_FILTER.RELEASES_ONLY)
+  const [versionFilter, setVersionFilter] = React.useState('')
```

Comment:
this will default to ocm-repo-cfg filter right? -> if so, please state in inline comment